src/utils/CanvasTemplates.js

```diff
@@ -720,7 +720,7 @@ module.exports = class CanvasTemplates {
     })
 
     //  IMAGES
-    const [ heartIcon, ...avatarImages ] = await IMAGE_ASSETS
+    const [heartIcon, ...avatarImages] = await IMAGE_ASSETS
```

Comment:
Why?

Reply:
All of these `Why?` modifications are Weebstorm autoformats. I haven't touched them. Shall I remove the modifications? @pedrofracassi

Reply:
Please do.